vocab_files/attribute_concepts/outlook-zone-right-direction.ttl

```diff
@@ -0,0 +1,17 @@
+PREFIX dcterms: <http://purl.org/dc/terms/>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX schema: <https://schema.org/>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX tern: <https://w3id.org/tern/ontologies/tern/>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+
+<https://linked.data.gov.au/def/nrm/a73f5249-542d-594f-b7d8-eb074c5405dc>
```

Comment:
```suggestion
<https://linked.data.gov.au/def/nrm/ea51c3ab-1d1e-48f7-ae83-8d06d4f97d04>
```
We need a new UUID here for a new concept

Reply:
Done